specification/compute/resource-manager/readme.go.md

```diff
@@ -35,6 +26,15 @@ output-folder: $(go-sdk-folder)/$(module-name)
 azure-arm: true
 
 directive:
+    # dynamically add a DummyOrchestrationServiceName value to the enum 
+  - from: compute.json
+    where: $..enum
+    transform: >-
+      if( $.length === 1 && $[0] === "AutomaticRepairs") { 
+        $.push('DummyOrchestrationServiceName');
```

Comment:
Do you know why this dummy value is required?

Reply:
Unfortunately I do not know. Maybe we should confirm this with the service team. Let me send some emails to ask

Reply:
It was added in https://github.com/Azure/azure-rest-api-specs/pull/8976 if that helps.